po/openpetfoodfacts/openpetfoodfacts.pot

```diff
@@ -80,7 +80,7 @@ msgid   "OpenFoodFacts"
 msgstr  ""
 
 msgctxt "warning_not_complete"
-msgid "This product page is not complete. You can help to complete it by editing it and adding more data from the photos we have, or by taking more photos using the app for <a href=\"https://play.google.com/store/apps/details?id=org.openpetfoodfacts.scanner\">Android</a> or <a href=\"https://apps.apple.com/us/app/open-beauty-facts/id1122926380\">iPhone/iPad</a>. Thank you!"
```

Comment:
@teolemon I don't think the site needs to know the language, it can just figure it out by itself.